src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ClientFileMetadataRouteSpec.scala

```diff
@@ -152,16 +153,21 @@ class ClientFileMetadataRouteSpec extends AnyFlatSpec with Matchers with TestReq
     ps.executeUpdate()
   }
 
-  private def checkClientFileMetadataExists(clientFileMetadataId: UUID): Unit = {
-    val sql = "select * from ClientFileMetadata where ClientFileMetadataId = ?;"
+  private def checkClientFileMetadataExists(fileId: UUID): Unit = {
+    val sql = "select * from FileMetadata where FileId = ? AND PropertyName IN (?,?,?,?);"
     val ps: PreparedStatement = DbConnection.db.source.createConnection().prepareStatement(sql)
-    ps.setString(1, clientFileMetadataId.toString)
+    ps.setString(1, fileId.toString)
+    clientSideProperties.zipWithIndex.foreach {
+      case (a, b) => ps.setString(b + 2, a)
+    }
     val rs: ResultSet = ps.executeQuery()
     rs.next()
-    rs.getString("ClientFileMetadataId") should equal(clientFileMetadataId.toString)
+    rs.getString("FileId") should equal(fileId.toString)
   }
 
   private def resetDatabase(): Unit = {
+    DbConnection.db.source.createConnection().prepareStatement("delete from FileMetadata").executeUpdate()
```

Comment:
Upper case commands "DELETE" and "FROM" for consistency?

Reply:
I've changed these as well.